libs/community/langchain_community/utilities/dria_index.py

```diff
@@ -0,0 +1,85 @@
+import logging
+from typing import Any, Dict, List, Optional, Union
+
+logger = logging.getLogger(__name__)
+
+
+class DriaAPIWrapper:
+    """Wrapper around Dria API.
+
+    This wrapper facilitates interactions with Dria's vector search
+    and retrieval services, including creating knowledge bases, inserting data,
+    and fetching search results.
+
+    Attributes:
+        api_key: Your API key for accessing Dria.
+        contract_id: The contract ID of the knowledge base to interact with.
+        top_n: Number of top results to fetch for a search.
+    """
+
+    def __init__(
+        self, api_key: str, contract_id: Optional[str] = None, top_n: int = 10
+    ):
+        from dria import Dria, Models
+
+        self.api_key = api_key
+        self.models = Models
+        self.contract_id = contract_id
+        self.top_n = top_n
+        self.dria_client = Dria(api_key=self.api_key)
+        if self.contract_id:
+            self.dria_client.set_contract(self.contract_id)
+
+    def create_knowledge_base(
+        self,
+        name: str,
+        description: str,
+        category: str = "Unspecified",
+        embedding: str = "jina-embeddings-v2-base-en",
```

Comment:
will users usually want to customize this? in that case may be clearer to not give it a default